pkg/security/scc/byrestrictions.go

```diff
@@ -31,20 +45,20 @@ func pointValue(constraint *securityapi.SecurityContextConstraints) int {
 	// strategies in order of least restrictive to most restrictive
 	switch constraint.SELinuxContext.Type {
 	case securityapi.SELinuxStrategyRunAsAny:
-		points += 4
+		points += runAsAnyUserWeight
```

Comment:
Yes, but creating a map, doing a lookup and checking a presence of the value will give us more code that wouldn't be so simple, short and easy to read. For such a simple case I believe it will be overkill.

Also did you mean that we should handle selinux and runAsUser strategies with a same mapping?

@pweil- WDYT?